store/indexer.ts

```diff
@@ -38,10 +39,15 @@ export const actions: ActionTree<IndexerState, IndexerState> = {
         data: { _metadata: data },
       }: any = await indexer
 
+      const lastProcessedTime = formatDistanceToNow(
+        new Date(Number(data?.lastProcessedTimestamp)),
+        { addSuffix: true }
+      )
+
       console.log(
         `%cIndexer:
         Health: ${data?.indexerHealthy ? '❤️' : '💀'}
-        Last: ${new Date(Number(data?.lastProcessedTimestamp))}`,
+        Last: ${lastProcessedTime}`,
```

Comment:
> Is it really necessary to have added date-fns import into indexer file? Does it fix something?

@roiLeo 
original string literal is `Date` type, so I replaced it with string message.